### DISCUSSION TITLE
Welcome to nestjs-boilerplate Discussions!

### DISCUSSION THREAD

**Discussion by vinayak25** [squareboat/nestjs-boilerplate#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Neosoulink** [comment]
Hi @vinayak25, I'm a full-stack software engineer and I would like to contribute to this NestJS project!

I'm not really good at presentations 😅, but if I can help I'll do it.
Is there any resource related to the project contribution?
